packit_service/worker/jobs.py

```diff
@@ -97,6 +102,47 @@ def process_jobs(self, event: Optional[Any]) -> Dict[str, HandlerResults]:
                     handler.clean()
         return handlers_results
 
+    def process_comment_jobs(self, event: Optional[Any]):
+        handlers_results = {}
+        # packit_command can be `/packit build` or `/packit build <with_args>`
+        (packit_mark, *packit_command) = event.comment.split(maxsplit=3)
```

Comment:
I'd catch ValueError here, just in case:
```
In [8]: a, *b = "".split(maxsplit=3)       
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-8-757e103c7539> in <module>
----> 1 a, *b = "".split(maxsplit=3)
        global a = 'a'
        global b.split = undefined
        global maxsplit = undefined

ValueError: not enough values to unpack (expected at least 1, got 0)
```
I guess that empty comments can't happen, but let's be prepared.

Reply:
Fixed by https://github.com/packit-service/packit-service/pull/89/commits/639e77d7449b0945c94aa5ef786207cda6f66db3